### ISSUE TITLE
Search stuck on "Initializing search" with Japanese search language and tags

### ISSUE THREAD

**Issue by tsreaper** [squidfunk/mkdocs-material#4469]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem

### Description

Following #4140. The search still stucks on "Initializing search".

I'm using the tags in my pages and uses Japanese as the search language (although the pages are written in Chinese). If I remove the tags or change the search language the issue disappears.

### Expected behaviour

Search works.

### Actual behaviour

Search stuck on "Initializing search".

### Steps to reproduce

1. Download this [reproducible example](https://www.dropbox.com/s/acrfqffd31467in/mkdocs-material-search-issue.zip?dl=0).
2. Use `mkdocs serve` to preview the site.

### Package versions

- Python: 3.8.10
- MkDocs: 1.4.0
- Material: 8.5.6

### Configuration

```yaml
site_name: My Docs

theme:
  name: material

plugins:
  - search:
      lang: ja
```

### System information

- Operating system: Windows 11, however I'm running python in WSL 2, with a Ubuntu 20.04.5 LTS image.
- Browser: Microsoft Edge 106.0.1370.37

**Comment by squidfunk** [comment]
Thanks for reporting. Duplicate of #4069. Please see https://github.com/squidfunk/mkdocs-material/issues/4069#issuecomment-1172851670 why fixing this is currently not economical.

**Comment by squidfunk** [comment]
The fix was released in 9.0.0.
